llvm/lib/TargetParser/Host.cpp

```diff
@@ -2082,8 +2082,13 @@ const StringMap<bool> sys::getHostCPUFeatures() {
                                    .Case("fp", "fp-armv8")
                                    .Case("crc32", "crc")
                                    .Case("atomics", "lse")
+                                   .Case("sha3", "sha3")
+                                   .Case("sm4", "sm4")
                                    .Case("sve", "sve")
                                    .Case("sve2", "sve2")
+                                   .Case("sveaes", "sve-aes")
+                                   .Case("svesha3", "sve-sha3")
+                                   .Case("svesm4", "sve2-sm4")
```

Comment:
As discussed offline, we should probably only enable `sve2-sm4` if SVE2 is also enabled to avoid defining the latter transitively.